### DISCUSSION TITLE
[Script request] Transmission update to V4

### DISCUSSION THREAD

**Discussion by burgerga** [community-scripts/ProxmoxVE#355] [Request script]
### Application Name

Transmission

### Website

https://transmissionbt.com/

### Description

Transmission is currently installed in Debian 12 but this is limited to V3 and it sometimes hangs (https://github.com/tteck/Proxmox/discussions/1719). There have been talks about backporting V4 to Debian 12, but so far no progress (https://github.com/transmission/transmission/discussions/5585#discussioncomment-7801385).
It's probably easier to move to Ubuntu 24.04 because it's readily available there (https://packages.ubuntu.com/noble/transmission)

(for completeness, the discussion in the tteck repo: https://github.com/tteck/Proxmox/discussions/1040)

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by azy2k9** [comment]
Can anyone support on this?

**Reply by tremor021** [reply]
I've created Alpine based script, because they are always up to date.

please test it:

`bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVED/refs/heads/main/ct/alpine-transmission.sh)"`

**Reply by burgerga** [reply]
Just a word of warning, there is an issue I'm having with Transmission V4 on a Ubuntu 24 LXC where after a while it hangs, I have currently "solved" it by restarting it at midnight using a cronjob, so I'm curious if you will encounter the same issue.

**Reply by brad-charboneau** [reply]
Even a bare Debian based LXC for Transmission 4 would be nice, especially if it can be upgraded in place over 3.0

I'm doubtful the script @tremor021 shared is setup for upgrading an existing transmission 3.0 to 4.0 yet

**Reply by burgerga** [reply]
Bumping the community script to debian 13 should solve this issue, because it has transmission 4.1-beta2

**Reply by MickLesk** [reply]
Feel free to bump Like the latest Scripts

**Reply by tremor021** [reply]
Wait, where does Transmission keeps its data? is it only in `/etc/transmission-daemon`?
